internal_metrics/metrics.yaml

```diff
@@ -0,0 +1,243 @@
+ebpf_net.agg_root_truncation:
+  brief:  Agg root truncation.
+  description: |
+    Total number of Aggregation root trunctions in Aggregation core of the reducer component.
+  title:  ebpf_net.agg_root_truncation
+
+ebpf_net.bpf_log:
+  brief: eBPF log count.
+  description: |
+     Total eBPF log count in the prior 30 seconds. bpf_logs are error messages from kernel-collector eBPF code that indicate unexpected or unhandled behavior. This is enabled by default.
+  metric_type: counter
+  title: ebpf_net.bpf_log
+
+ebpf_net.client_handle_pool:
+  brief: Client handle pool in the prior 30 seconds.
+  description: |
+    Client handle pool for the prior thirty seconds.
+  metric_type: counte
+  title: ebpf_net.client_handle_pool
+
+ebpf_net.client_handle_pool_fraction:
+  brief:  Client handle pool fraction.
+  description: |
+    Client handle pool fraction.
+  metric_type: counter
+  title:  ebpf_net.client_handle_pool_fraction
+
+ebpf_net.clock_offset_ns:
+  brief:  Clock offset in ns.
+  description: |
+    Clock offset in nanoseconds.
+  metric_type: counter
+  title:  ebpf_net.clock_offset_ns
+
+ebpf_net.codetiming_avg_ns:
+  brief: The average code timing time in ns.
+  description: |
+    The average time for a particular instrumented code block in nanoseconds.
+  metric_type: counter
+  title: ebpf_net.codetiming_avg_ns
+
+ebpf_net.codetiming_count:
+  brief: The number of codetiming blocks.
+  description: |
+     The number of codetiming blocks.
+  metric_type: counter
+  title: ebpf_net.codetiming_count
+
+ebpf_net.codetiming_max_ns:
+  brief: Maximum code timing in nanoseconds.
+  description: |
+    Maximum code timing in nanoseconds.
+  metric_type: counter
+  title: ebpf_net.codetiming_max_ns
+
+ebpf_net.codetiming_min_ns:
+  brief: The minimum code timing in the prior 30 seconds.
+  description: |
+    The minimum code timing for an instrumented block of code for the prior 30 seconds.
+  metric_type: counter
+  title: ebpf_net.codetiming_min_ns
+
+ebpf_net.codetiming_sum_ns:
+  brief: Code timing sum in nanoseconds.
+  description: |
+    Code timing sum in nanoseconds.
+  metric_type: counter
+  title: ebpf_net.codetiming_sum_ns
+
+ebpf_net.collector_health:
+  brief: Collector health status
+  description: |
+    Collector health status. This metrics shows if Network Explorer collectors are up and running to properly extract information using at the kernel level. This is enabled by default.
```

Comment:
Do we want to note what 'healthy value' is?  I seem to recall some team conversation that this should be '0'?

Reply:
noted.